bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/handlers/PythonCleanCommandHandler.java

```diff
@@ -0,0 +1,79 @@
+/*******************************************************************************
+ * Copyright 2021 Espressif Systems (Shanghai) PTE LTD. All rights reserved.
+ * Use is subject to license terms.
+ *******************************************************************************/
+
+package com.espressif.idf.ui.handlers;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import org.eclipse.core.commands.ExecutionEvent;
+import org.eclipse.core.commands.ExecutionException;
+import org.eclipse.core.resources.IProject;
+import org.eclipse.core.runtime.IStatus;
+import org.eclipse.core.runtime.Path;
+
+import com.espressif.idf.core.IDFCorePlugin;
+import com.espressif.idf.core.IDFEnvironmentVariables;
+import com.espressif.idf.core.ProcessBuilderFactory;
+import com.espressif.idf.core.logging.Logger;
+import com.espressif.idf.core.util.IDFUtil;
+import com.espressif.idf.ui.EclipseUtil;
+import com.espressif.idf.ui.update.AbstractToolsHandler;
+
+public class PythonCleanCommandHandler extends AbstractToolsHandler
+{
+	@Override
+	public Object execute(ExecutionEvent event) throws ExecutionException
+	{
+		activateIDFConsoleView();
+		if (!NewProjectHandlerUtil.installToolsCheck())
+		{
+			return null;
+		}
+		IProject selectedProject = EclipseUtil.getSelectedProjectInExplorer();
+
+		Path pathToProject = new Path(selectedProject.getLocation().toString());
+		List<String> commands = new ArrayList<>();
+		commands.add(IDFUtil.getIDFPythonEnvPath());
+		commands.add(IDFUtil.getIDFPythonScriptFile().getAbsolutePath());
+		commands.add("clean"); //$NON-NLS-1$
+		Map<String, String> envMap = new IDFEnvironmentVariables().getEnvMap();
+		console.print((runCommand(commands, pathToProject, envMap)));
+		return event;
+	}
+
+	private String runCommand(List<String> arguments, Path workDir, Map<String, String> env)
+	{
+		String exportCmdOp = ""; //$NON-NLS-1$
+		ProcessBuilderFactory processRunner = new ProcessBuilderFactory();
+		try
+		{
+			IStatus status = processRunner.runInBackground(arguments,
+					workDir, env);
+			if (status == null)
+			{
+				Logger.log(IDFCorePlugin.getPlugin(), IDFCorePlugin.errorStatus("Status can't be null", null)); //$NON-NLS-1$
+
```

Comment:
Return an error message if the status is null here

Reply:
@kolipakakondal done